habitat_baselines/common/obs_transformers.py

```diff
@@ -468,29 +852,8 @@ def __init__(
         cx, cy: (float) the optical center of the generated fisheye
         fx, fy, xi, alpha: (float) the fisheye camera model parameters
         """
-        super(Cube2Fisheye, self).__init__()
-        self.fish_h = fish_h
-        self.fish_w = fish_w
-        self.fish_fov = fish_fov
-        self.fish_param = [cx, cy, fx, fy, xi, alpha]
-        self.grids = self.generate_grid(
-            fish_h, fish_w, fish_fov, self.fish_param
-        )
-        self._grids_cache = None
 
-    def generate_grid(
-        self,
-        fish_h: int,
-        fish_w: int,
-        fish_fov: float,
-        fish_param: List[float],
-    ) -> torch.Tensor:
-        # Project on sphere
-        xyz_on_sphere, fov_mask = self.get_points_on_sphere(
-            fish_h, fish_w, fish_fov, fish_param
-        )
-
-        # Rotate so that each face will be in front of camera
+        # Cubemap input
         rotations = [
```

Comment:
We keep reusing this ordering so maybe it should be a constant/enum somewhere too? Also any reasons its np.arrays instead of a native PyTorch array? It could just be one multidimensional torch.Tensor after all instead of list of np.arrays

Reply:
Shouldn't the RotationMat class at least inherit from 'Enum' then? https://docs.python.org/3/library/enum.html